packages/fluentui/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -129,14 +128,30 @@ const alertStyles: ComponentSlotStylesPrepared<AlertProps, AlertVariables> = {
       ...getIconFillOrOutlineStyles({ outline: true }),
 
       ':hover': {
+        backgroundColor: v.hoverBackgroundColor,
         color: 'currentColor',
-        ...iconFilledStyles
+        ...iconFilledStyles,
+        // TODO: consider creating dedicated method for border styles on hover
+        ...getBorderFocusStyles({
+          variables: {
+            borderRadius: v.focusHoverBorderRadius,
+            borderWidth: v.focusHoverBorderWidth,
+            focusInnerBorderColor: v.focusHoverInnerBorderColor,
+            focusOuterBorderColor: v.focusHoverOuterBorderColor,
+            zIndexes: { foreground: v.focusHoverZIndex }
+          }
+        })[':focus-visible']
       },
 
       ':focus': borderFocusStyles[':focus'],
       ':focus-visible': {
+        backgroundColor: v.focusBackgroundColor,
         ...iconFilledStyles,
-        ...borderFocusStyles[':focus-visible']
+        ...borderFocusStyles[':focus-visible'],
+
+        ':hover': {
```

Comment:
This is really confusing, we have first in the file:
```
':hover': {
   ...getBorderStyles({...})[':focus-visible']
}
```
and than:
```
':focusVisible': {
  // ...
  ':hover': { ... }
}
```
and we have the backgroundColor in both of them.. I have no idea which one will win and why...

Reply:
Should we always define these in some order?

Reply:
Oops, it had a sense before when we have had different values for `hoverBackgroundColor` & `focusBackgroundColor`. Now it can be removed at 🤔

Reply:
I removed `:hover` as there is no sense in it